src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,60 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book testBook1 = bookDao.create(new Book("Harry Potter and the Philosophers Stone",
+                BigDecimal.valueOf(9.99)));
+
+        //findById method test
+        Optional<Book> testFindBook = bookDao.findById(1L);
+        if (testFindBook.isPresent()) {
+            Book test1 = testFindBook.get();
+            System.out.println("Find by ID1 book: " + System.lineSeparator() + test1);
```

Comment:
When retrieving a book by ID, you're assuming the ID '1' will always exist. This may not be the case, especially if this code is run multiple times or if the database is modified externally. It's better to check if `testBook1` was created successfully and use its generated ID to find it again.